php_1/db/PdoConnectionClass.php

```diff
@@ -1,16 +1,25 @@
 <?php
+require_once 'Config.php';
+
 class PdoConnectionClass
 {
-    private $server = "mysql:host=localhost;dbname=mydatabase3";
-    private $user = "root";
-    private $pass = "";
+    private $serverName = Config::HOST;
+    private $database = Config::DATABASE;
+    private $user = Config::USER;
+    private $pass = Config::PASSWORD;
     private $options = array(PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION, PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC);
     protected $con;
+    protected $db;
 
-    public function connect()
+    public function __construct()
+    {
+        $this->db = $this->connect();
+    }
+
+    private function connect()
     {
         try {
-            $this->con = new PDO($this->server, $this->user, $this->pass, $this->options);
+            $this->con = new PDO("mysql:host=$this->serverName;dbname=$this->database", $this->user, $this->pass, $this->options);
             return $this->con;
```

Comment:
we can delete this line as $this->con as it is set one line above. After remove we can set the connect function return typeas: void